odoo/tests/common.py

```diff
@@ -1596,6 +1598,10 @@ def setUp(self):
             self.registry.enter_test_mode(self.cr)
             self.addCleanup(self.registry.leave_test_mode)
 
+        for fn in self.test_setups:
+            with suppress(Exception):
```

Comment:
why do you need the supress? looks like a good way to have a broken one never detected.

Reply:
I thought one setup failing should not prevent the others to run. But you are right, if something breaks we want the test to fail and the error to be raised. I'll update this part.